nbs/00_betting_env.ipynb

```diff
@@ -0,0 +1,1565 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    module_path = os.path.abspath(os.path.join(".."))
</pre><p>Same issue discussed in index nb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MeherKharbachi/betting_env/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>